### DISCUSSION TITLE
Github and PCI 4.0

### DISCUSSION THREAD

**Discussion by karl-dpg** [community/community#112333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

With PCI 4.0, Github will become in-scope for auditing and with that, we need to provide an AoC if we want to pass our audit.

Is Github already providing this document? If not, are there any plans for Github to become PCI compliant?

**Comment by karl-dpg** [comment]
Any update on this please? As things stand, we are not allowed to use Github.com.

**Comment by codonnell-pt** [comment]
I am also looking for an answer to this. Anyone know of the status of this ?
